docs/docs/model-storage.mdx

```diff
@@ -2,156 +2,214 @@
 
 You can configure the HTTP server to fetch models from another URL
-by adding it to your `endpoints.yml`. The server will then
-reguarly query this URL for zipped models.
+by adding it to your `endpoints.yml`:
 
 ```yaml-rasa title="endpoints.yml"
 models:
-  url: http://my-server.com/models/default@latest
+  url: http://my-server.com/models/default
   wait_time_between_pulls: 10   # In seconds, optional, default: 100
 ```
+The server will query `url` for a zipped model every `wait_time_between_pulls`
+seconds.
 
-:::note
-If you want to pull the model just once from the server, set
-`wait_time_between_pulls` to `None`.
-
-:::
-
-:::note
-Your model server must provide zipped Rasa models, and have
-`{"ETag": <model_hash_string>}` as one of its headers. Rasa will
-only download a new model if this model hash has changed.
+If you want to pull the model just once from the server, you can set the time
+between pulls to `null`:
 
-:::
+```yaml-rasa title="endpoints.yml"
+models:
+  url: http://my-server.com/models/default
+  wait_time_between_pulls: null  # fetches model only once
+```
 
-Rasa sends requests to your model server with an `If-None-Match`
-header that contains the current model hash. If your model server can
-provide a model with a different hash from the one you sent, it should send it
-in as a zip file with an `ETag` header containing the new hash. If not, Rasa
-expects an empty response with a `204` or `304` status code.
+#### Configuration of your server
 
-An example request Rasa might make to your model server looks like this:
+Rasa Open Source will send a `GET` request to the URL you specified in the
+`endpoints.yml`, e.g. in the above examples this would be
+`http://my-server.com/models/default`. You can use any URL.
+The `GET` request will contain an `If-None-Match` header that contains the
+model hash of the last model it downloaded. An example request from Rasa Open
+Source to your server would look like this:
 
 ```bash
-curl --header "If-None-Match: d41d8cd98f00b204e9800998ecf8427e" http://my-server.com/models/default@latest
+curl --header "If-None-Match: d41d8cd98f00b204e9800998ecf8427e" http://my-server.com/models/default
 ```
 
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="server-fetch-from-remote-storage"></a>
+The response of your server to this `GET` request should be one of these:
+- return a status code of `200`, a zipped Rasa Model and set the `ETag` header in
+  the response to the hash of the model.
+- return a status code of `304` and an empty response if the `If-None-Match`
+  header of the request matches the model you want your server to return.
+
+Rasa Open Source uses the `If-None-Match` and `ETag` headers fro caching. Setting
+the headers will avoid re-downloading the same model over and over, saving
+bandwidth and compute resources.
 
-## Fetching Models from Remote Storage
+An example request Rasa might make to your model server looks like this:
```

Comment:
?

Reply:
just a leftover from the previous version, the example as already further up 🙂